src/OrbitGl/CaptureWindow.cpp

```diff
@@ -424,7 +424,11 @@ void CaptureWindow::KeyPressed(unsigned int key_code, bool ctrl, bool shift, boo
       time_graph_->GetTrackContainer()->IncrementVerticalScroll(
           /*ratio=*/-kScrollingRatioPerPageUpAndDown);
       break;
+    // Adding 93 == ']' here such that with e.g. German and Italian keyboards the ctrl+"+" shortcut
+    // works. As an unwanted side effect ctrl+"]" on an English keyboard also acts as vertical zoom.
+    // See b/237773876.
     case '+':
+    case 93:
```

Comment:
Can I ask you to also add '=' as discussed online, so Ctrl + '+' works without needed to press shift? 

Context: https://doc.qt.io/qt-5/qkeysequence.html#keyboard-layout-issues

Reply:
Sg, added "=", "_" as discussed.